src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -22,8 +22,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuildWebTargetsPath>$(MSBuildExtensionsPath)\Microsoft\VisualStudio\Managed.Web\</MSBuildWebTargetsPath>
     <AspNetCoreHostingModel Condition="'$(AspNetCoreHostingModel)' == '' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' != '' And '$(_TargetFrameworkVersionWithoutV)' &gt;= '3.0' ">inprocess</AspNetCoreHostingModel>
     <AspNetCoreModuleName Condition="'$(AspNetCoreModuleName)' == '' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' != '' And '$(_TargetFrameworkVersionWithoutV)' &gt;= '3.0' ">AspNetCoreModuleV2</AspNetCoreModuleName>
+    <EnableAspNetCoreGlobalUsings Condition="
```

Comment:
@vijayrkn I don't see any tests for the WebSDK. I've been doing spot checks with https://github.com/JunTaoLuo/GlobalUsings. Is that sufficient to verify this change?

Reply:
There are unit tests for the tasks & end-end tests for the folder publish. Let me check the status of those.

Reply:
@JunTaoLuo  - I think we need to add something similar to this https://github.com/dotnet/sdk/blob/main/src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/EndToEnd/FolderPublish50.cs but rather than publish, it should cover 'dotnet run' for web.

Reply:
Tests added